.github/workflows/test-integration.yml

```diff
@@ -0,0 +1,42 @@
+name: Test Integration
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Comment:
I guess we should run tests for each PR

Reply:
On each PR to master  .. but that may be why those aren't running since in this repo it's `main`. So we should adapt this.
I hope this isn't the reason why the other workflow isn't triggered either?